.github/dependabot.yml

```
@@ -0,0 +1,7 @@
+version: 2
+updates:
+  - package-ecosystem: "cargo"
+    directory: "/rust"
+    labels: ["dependabot"]
+    schedule:
+      interval: "daily"
```

```suggestion
      interval: "daily"
    - package-ecosystem: github-actions
      directory: '/'
      labels: ["dependabot"]
      schedule:
        interval: daily
```

Also it would be great to add GitHub Action check.